src/utils/DragAndDrop.js

```diff
@@ -183,6 +183,10 @@ define(function (require, exports, module) {
             event.preventDefault();
 
             options.ondragleave(event);
+            // adding functionality when dragging files away from FILE bar
```

Comment:
Question: why are we adding these inside the event handlers?  Why aren't we just adding it once above when we initialize all the listeners?  @flukeout when you test this, can you look at this too?  It seems wrong to me, and like it will just add multiple event handlers we don't want.